momentum/4/modules/ha-proxy-client.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "ha_proxy_client - HAProxy protocol client module"
+description: "The ha_proxy_client module is used to configure Momentum to use HAProxy's PROXY protocol version 2 for outbound connections (see https://github.com/haproxy/haproxy/blob/master/doc/proxy-protocol.txt).  This can be leveraged in cases where your sending IPs are deployed on a different machine than the Momentum.
+---
+
+When configured for a binding or binding group Momentum will connect to the given `ha_proxy_server` and prefix the SMTP session with a PROXY protocol version 2 header.  The `dst_addr` and `dst_port` will be filled in with the resolved MX, the `src_addr` will be filled in with the configured value of `ha_proxy_src_addr`.  It is the customers responsibility to configure a listener on `dst_addr:dst_port` that listens for PROXY protocol and routes based on the `src_addr`, but a simple example configuration for HAProxy would be:
+
+```
+# listen to proxy protocol and forward to backend
```

Comment:
haven't actually tested this config but pulled from https://sparkpost.atlassian.net/wiki/spaces/ENG/pages/167509374/R%2BD%2BTesting%2BOutbound%2BProxy%2BSupport%2Bin%2BPowerMTA

will test once merged